views/pages/addTechnology.ejs

```diff
@@ -89,7 +93,7 @@
 <script>
 
     $("form").submit(function (event) {
-
+        $(":submit").attr("disabled", true);
```

Comment:
I can't find a case in which the button stays disabled. When an error occurs, the button gets activated anyway. When should it remain inactive?